cpp/tests/io/parquet_test.cpp

```diff
@@ -3198,4 +3204,26 @@ TEST_F(ParquetWriterTest, RowGroupSizeInvalid)
                cudf::logic_error);
 }
 
+TYPED_TEST(ParquetWriterSchemaTest, FieldID)
```

Comment:
we should now be able to test that IDs can survive a round trip through a Parquet file.

Reply:
Currently bypass the round trip test since the reader-side metadata is not exposed.